x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -52,6 +53,22 @@ export function ServiceOverview() {
   const { serviceName, fallbackToTransactions, agentName, serverlessType } =
     useApmServiceContext();
 
+  const { setScreenContext } =
+    useApmPluginContext().observabilityAIAssistant.service;
+
+  useEffect(() => {
+    return setScreenContext({
+      screenDescription: `The user is looking at the service overview page for ${serviceName}.`,
+      data: [
+        {
+          name: 'service_name',
+          description: 'The name of the service',
+          value: serviceName,
+        },
+      ],
+    });
+  }, [setScreenContext, serviceName]);
```

Comment:
This might not be needed. Yet, I expect it to improve the LLMs confidence in determining what the current service is.